.github/workflows/markdown-linting/.markdownlint.json

```diff
@@ -0,0 +1,11 @@
+{
+  "no-inline-html": false,
```

Comment:
These rules weren't compatible with how we're using the `md` files currently, or were blocking the PR from going live, so I've disabled them for now.

@kachkaev you can give these a look as well. We can fix the files manually based on the rules we enable in a follow-up task.

Reply:
Starting with a small number of rules and then extending it is a good strategy, so happy for us to disable rules that would make git diff too massive 👍